contracts/BoundaryManager.sol

```diff
@@ -59,6 +59,11 @@ contract BoundaryManager {
         address prevOwner = address(0);
         unchecked {
             for (uint16 i = 0; i < safeRefundBoundary.allowedRefundReceiversCount; i++) {
+                
+                // COMMENT: will this handle the case when there are 2 duplicate addresses?
+                // refundReciever[i] = 1 and prevOwner = 1 should pass this <if statement>'s conditional
```

Comment:
but `1 < 1` is false isn't it?

Reply:
this is the same thing that the safe contract does when checking signatures to prevent counting signatures from the same owner twice

Reply:
As I understand it, because "1 < 1" is false, the "revert" will not execute, that means that you still have a duplicate address and it is not reverted.

Reply:
Safe uses require, you should therefore invert what the Safe does, which would include the equal

Reply:
ahh sorry brainfart gentlemen, thanks for the catch